Graphics/GraphicsEngineD3DBase/include/EngineFactoryD3DBase.hpp

```diff
@@ -300,7 +305,7 @@ class EngineFactoryD3DBase : public EngineFactoryBase<BaseInterface>
     }
 
 private:
-    template <RENDER_DEVICE_TYPE DevType>
+    template <RENDER_DEVICE_TYPE MyDevType>
```

Comment:
Is this really needed?

Reply:
Clang generates a name shadowing warning otherwise.